racket/src/ChezScheme/c/Mf-arm64osx

```diff
@@ -39,7 +39,7 @@ ${Scheme}: ${Kernel} ${KernelLinkDeps} ${Main}
 	$C -o ${Scheme} ${Main} ${Kernel} ${KernelLinkLibs} ${LDFLAGS} ${LIBS}
 
 ../zlib/configure.log:
-	(cd ../zlib; ${SetConfigEnv} CFLAGS="${CFLAGS}" ./configure)
+	(cd ../zlib; ${SetConfigEnv} CPPFLAGS="${CPPFLAGS}" CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}" ./configure)
```

Comment:
The Racket makefile currently folds `CPPFLAGS` into `CFLAGS` to pass to the Chez Scheme makefiles — and so on to the zlib and lz4 makefiles — because distinguishing them is a pain, and I don't think CS will need the distinction in the long run that BC needed. Does combing them into `CFLAGS` not work here, though?